extensions/script-libraries/semver/semver.livecodescript

```diff
@@ -0,0 +1,722 @@
+﻿script "semver"
```

Comment:
Ah, yes I hadn't thought about that... good plan as it fits with my thoughts on packages